statsmodels/tsa/stattools.py

```diff
@@ -703,14 +754,22 @@ def pacf(x, nlags=40, method='ywunbiased', alpha=None):
     confint : array, optional
         Confidence intervals for the PACF. Returned if confint is not None.
 
+    See also
+    --------
+    statsmodels.tsa.stattools.acf
+    statsmodels.tsa.stattools.pacf_yw
+    statsmodels.tsa.stattools.pacf_burg
+    statsmodels.tsa.stattools.pacf_ols
+
     Notes
     -----
     This solves yule_walker equations or ols for each desired lag
     and contains currently duplicate calculations.
     """
 
-    if method == 'ols':
-        ret = pacf_ols(x, nlags=nlags)
+    if method.startswith('ols'):
```

Comment:
Do we want to deny invalid strings starting with "ols"?

Reply:
I changed this